v2/src/App.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import './App.css';
+import { ActiveTransit, TransitIconImages } from './Interfaces';
+import logo from './assets/trajectory_logo.svg';
+import buttonDown from './assets/button-down.svg';
+import GOTrainLight from './assets/gotrain-light.svg';
+import VIALight from './assets/via-light.svg';
+import StreetCarLight from './assets/streetcar-light.svg';
+import SubwayLight from './assets/subway-light.svg';
+import BusLight from './assets/bus-light.svg';
+import searchLight from './assets/search-light.svg';
+
+const samepleVehicles = [
+  {
+    type: 'bus',
+    route: 505,
+    direction: 'Westbound',
+    terminal: `Queen's Park`,
+    delay: 5
+  },
+  {
+    type: 'streetcar',
+    route: 105,
+    direction: 'Northound',
+    terminal: `Queen's Park`,
+    delay: 10
+  },
+]
+
+const transitImagesLight: TransitIconImages = {
+  gotrain: GOTrainLight,
+  via: VIALight,
+  streetcar: StreetCarLight,
+  subway: SubwayLight,
+  bus: BusLight
+}
+
+const App = () => {
+  const [dropdown, activateDropdown] = useState(false);
+  const [search, activateSearch] = useState(false);
+  const [activeTransit, setActiveTransit] = useState<ActiveTransit>({
+    gotrain: true,
+    via: true,
+    streetcar: false,
+    subway: true,
+    bus: false
+  });
+  const [currentVehicles, setCurrentVehiclces] = useState([]);
+  const [searchValue, setSearchValue] = useState('');
+  const [searchSuggestions, setSearchSuggestions] = useState([]);
+  
+  const transitIcon = (transitType: string) => (
+    <div 
+      key={transitType} 
+      onClick={() => setActiveTransit({...activeTransit, [`${transitType}`]: !activeTransit[transitType]})} 
+      className={`transit-icon-container ${transitType} ${!activeTransit[transitType] && 'inactive'}`}
+    >
+      <img className="transit-icon" src={transitImagesLight[transitType]} alt={`${transitType} icon`} />
+    </div>
+  );
+  
+  const transitOption = (title: string, transitType: string, icon: string) => (
+    <div className="transit-box">
+      {transitIcon(transitType)}
+      <h3 className="transit-header">
+        {title}
+      </h3>
+    </div>
+  );
+
+  return (
+    <div className="main-container">
+      <div className="map">
+      </div>
+      <div className="controls">
+        <div className="side-menu-container">
+          <div className="side-menu-header-container">
+            <h2 className="side-menu-header">
+              Active transit
+            </h2>
+            <img 
+              onClick={() => activateDropdown(!dropdown)} 
+              className={`side-menu-button ${dropdown ? 'dropdown-on' : 'dropdown-off'}`} 
```

Comment:
I highly recommend installing and using `clsx`

Reply:
Done on all cases 👍